content/transfers.html

```diff
@@ -0,0 +1,69 @@
+<h1 id="transfers"> Transfers </h1>
+<h3> Programs </h3>
+<p class="half">
+<span class="big">
+    All students who transferred out of SE, went into the Computer Science Program.
+</span>
+Of the people surveyed approximately 9.5% of students switched out of the Software Engineering Program.
+</p>
+<div class="half second">
+    <span class="graph-title">Proportion of students who transferred.</span>
+    <div id="students-transferred"></div>
+</div>
+
+<div class="vbx2"></div>
+
+<h3> Term Switched Out </h3>
+<p class="half">
+<span class="big">
+    Students started switching out of the program after completing second year
```

Comment:
I think the reason why we are not seeing any time earlier is that we only asked people who at least spent 2 years in the program to complete the survey. Maybe we should incorporate the transfer info from the 2019 se board too for a more complete picture https://patricklam.ca/se/se-board-2019-dec/